site/docs/src/content/docs/start/virtualmachine.mdx

```diff
@@ -130,39 +137,34 @@ sudo camblet --config /etc/camblet/config.yaml agent generate-policy 5241 nginx
     process:uid: "33"
 ```
 
-Camblet will use these selectors to identify the nginx server. The connection part configures the mTLS. Since it is strict, only clients with certificates can communicate with it. To verify that, let's try it with cURL.
+> The saved policy file is automatically read by the running Camblet agent and gets updated within the kernel.
 
-### Try to connect to nginx without a certificate
+Camblet will use the `selectors` to identify the NGINX server. The `connection` part configures the TLS settings where `STRICT` mTLS value means only clients with trusted certificates can communicate with it.
 
-Try to connect to nginx on localhost:80:
+To verify that, let's try connecting to NGINX with cURL.
 
 ```sh
-curl localhost:80
-```
+$ curl localhost
 
-As we waited, we received the following:
-
-```sh
-curl localhost:80
-curl: (52) Empty reply from server
+curl: (56) Recv failure: Connection reset by peer
 ```
 
-### Create policy for the curl command
+### Create policy for cURL
 
-To create a new policy for cURL to communicate with the server, we need the PID of cURL. 
-Since cURL is not running continuously like the server, we have to use a "dummy" command to force it to run as long as we generate the policy. 
-
-`This must be done before cURL times out; otherwise, the PID changes.`
+To create a new policy for cURL to communicate with the server, we need the PID of cURL. Since cURL is not running continuously like the server, a "dummy" command is needed to force it to run until generate the policy.
```

Comment:
```suggestion
To create a new policy for cURL to communicate with the server, we need the PID of cURL. Since cURL is not running continuously like the server, a "dummy" command is needed to force it to run until the policy is generated.
```